docs/calico-with-docker/AWS.md

```diff
@@ -283,38 +288,39 @@ curl http://<host public ip>:80
 
 # (Optional) Modifying MTU for Performance Boost
 
-Some [AWS instance types](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/network_mtu.html#jumbo_frame_instances) 
-utilize a default [MTU](https://en.wikipedia.org/wiki/Maximum_transmission_unit) 
-of 9001, which is larger than the standard 1500 used by most of the Internet. 
+Some [AWS instance types](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/network_mtu.html#jumbo_frame_instances)
+utilize a default [MTU](https://en.wikipedia.org/wiki/Maximum_transmission_unit)
+of 9001, which is larger than the standard 1500 used by most of the Internet.
 
-If you have a high traffic deployment and all of your host instances in your 
-AWS Calico cluster are using these instance types, you may be able to improve 
-performance for traffic between your Calico nodes. To do this, you need to 
-modify the MTU of the IP-in-IP tunnels on your Calico hosts and the veth 
-interfaces of your Calico containers.
+If you have a high traffic deployment and all of your host instances in your
+AWS Calico cluster are using these instance types, you may be able to improve
+performance for traffic between your Calico nodes. To do this, you need to
+modify the MTU of the veth interfaces of your Calico containers.
 
-> **WARNING**: You must run these commands on ALL of the hosts and containers in your 
-> deployment. If not all of your instances are jumbo frame instances or if you 
-> do not modify MTU on all containers and hosts, you may  experience unexpected 
+> **WARNING**: You must run these commands on ALL of the hosts and containers in your
+> deployment. If not all of your instances are jumbo frame instances or if you
+> do not modify MTU on all containers and hosts, you may  experience unexpected
 > behavior and/or packet loss.
 
 #### Modify Calico interface MTU on hosts
-First, modify the MTU on the Calico tunnel interface and `cali######` 
-interfaces on your host instances (use `ip link` or `ifconfig` to see the 
+First, modify the MTU on the `cali######`
+interfaces on your host instances (use `ip link` or `ifconfig` to see the
 interface names):
 ```
-sudo ifconfig <interface> mtu 8981
+sudo ifconfig <interface> mtu 9001
 ```
 Repeat for all relevant Calico interfaces on the host.
 
-Note that the value used is 8981.  This is because Calico traffic will flow 
-through the IP-in-IP tunnel.  When packets enter the IP-in-IP tunnel, an IP 
+If you're using IP-in-IP, instead use a value of 8981 throughout.
+This is because Calico traffic will flow
+through the IP-in-IP tunnel.  When packets enter the IP-in-IP tunnel, an IP
 header of length 20 is added to the packet, summing to a total size of 9001.
+You'll also need to set the MTU of the `tunl0` interface to 8981.
```

Comment:
This is probably going away in the next couple weeks, but for simplicity just add this to the line above:

"Repeat for all relevant Calico interfaces and the `tunl0` interface on the host."